flower/utils/broker.py

```diff
@@ -168,13 +172,35 @@ def __init__(self, broker_url, *args, **kwargs):
                                  password=self.password)
 
 
+class RedisSsl(Redis):
+    """
+    Redis SSL class offering connection to the broker over SSL.
+    This does not currently support SSL settings through the url, only through
+    the broker_use_ssl celery configuration.
+    """
+
+    def __init__(self, broker_url, *args, **kwargs):
+        if 'broker_use_ssl' not in kwargs:
+            raise ValueError('rediss broker requires broker_use_ssl')
+        self.ssl_cert_reqs = kwargs['broker_use_ssl']['ssl_cert_reqs']
+        super(RedisSsl, self).__init__(broker_url, *args, **kwargs)
+
+    def _get_redis_client_args(self):
+        client_args = super(RedisSsl, self)._get_redis_client_args()
+        client_args['ssl'] = True
+        client_args['ssl_cert_reqs'] = self.ssl_cert_reqs
```

Comment:
All the configuration options to celery map to those of redis-py. So just return the `broker_use_ssl` and updated with ssl=True

Reply:
good point! I updated this to just pass through everything from `broker_use_ssl`.